gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/ManifestBasedDataset.java

```diff
@@ -117,6 +126,7 @@ public Iterator<FileSet<CopyEntity>> getFileSetIterator(FileSystem targetFs, Cop
         CommitStep step = new DeleteFileCommitStep(targetFs, toDelete, this.properties, Optional.<Path>absent());
         copyEntities.add(new PrePublishStep(datasetURN(), Maps.newHashMap(), step, 1));
       }
+      log.info(String.format("Workunits calculation take %s milliseconds to process %s files", System.currentTimeMillis() - startTime, numFiles));
```

Comment:
nit: "take" => "took"

BTW, I really like this timing!  do we want a metric in addition to a logged line?

Reply:
We already have an event emitted for the workunit creation time (event name isWorkUnitsCreationTimer). I add a log here just for good observability for a single benchmark job.